core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/termination/StackTerminationAction.java

```diff
@@ -59,7 +58,6 @@ protected void doExecute(StackTerminationContext context, TerminateStackRequest<
             TerminateStackResult terminateStackResult = new TerminateStackResult(statusReason, new IllegalArgumentException(statusReason), terminateRequest);
             sendEvent(context.getFlowId(), StackTerminationEvent.TERMINATION_FAILED_EVENT.stringRepresentation(), terminateStackResult);
         } else {
-            MDCBuilder.buildMdcContext(stack);
```

Comment:
Why was it deleted? The common fields of log messages on this thread won't be filled.